index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Porfolio Website</title>
+  </head>
+  <body>
+    <header id="nav-bar">
+      <div id="logo">
+        <h4>Said Laasri</h4>
+      </div>
+      <nav id="side-bar">
+        <div id="victor1"></div>
+        <div id="victor2"></div>
+        <div id="victor3"></div>
+      </nav>
+    </header>
+    <section id="header">
+      <h1>Hey There.i'm Said</h1>
+      <h3>I am a Software Developer</h3>
```

Comment:
- [ ] Please replace `<h3>` with `<h2>`.  Kindly note this: Do not skip heading levels. Use the `<h1>` as the main headings of the webpage, followed by the `<h2>` headings, then the less important `<h3>` headings, etc.

Reply:
@OLIPLICHE Thank you for reviewing my project, I will make the necessary corrections as suggested